composer.json

```diff
@@ -54,6 +88,7 @@
         "drupal/xmlsitemap": "~1.0@alpha",
         "ext-imagick": "*",
         "su-sws/jumpstart_ui": "dev-8.x-1.x",
+        "su-sws/nobots": "dev-8.x-1.x",
```

Comment:
Does this profile rely on this module? Perhaps the stack does but not this install profile.

Reply:
anything in core.extension.yml has to be added as a dependency for testing

Reply:
When did that get in there? Ok thanks.

Reply:
probably BLT https://github.com/SU-SWS/acsf-cardinalsites/blob/8.x-1.x/blt/blt.yml#L31